### PR TITLE
Fix for missing icons: Require Qt SVG plug-in

### PR DESCRIPTION
**Pull request by shundhammer**
## Bugzilla

https://bugzilla.suse.com/show_bug.cgi?id=1114654

## Trello

https://trello.com/c/27ozQgUS/452-ostumbleweed-p1-1114654-build-20181102-patternselector-icons-missing-openqa-test-fails-in-selectpatternsandpackages

## Problem

No status icons in the patterns or package selector in Qt, in the inst-sys as well as in the installed system later.

## Cause

No support for SVG available. We have compiled-in fallback icons, but they are also SVG now, so they cannot be rendered or maybe even not loaded.

In previous versions, the libyui-qt-pkg plugin was linked against libqt5svg.so (it was listed in the `ldd`) output, but since this current Tumbleweed snapshot it is no longer. Why this is so is not completely clear; very likely it was an indirect dependency where some other part of Qt was using it. Even trying to force linking it to libyui-qt-pkg failed every time; we are not actively using it, only indirectly, and it is a plug-in to the Qt image loading subsystem; so the linker saw our link instruction and still decided that it is not necessary and simply did not link that lib to our binary (I tried different approaches with that).

## Solution

Add a plain and simple explicit `Requires` for the package (not the library) to our .spec file.

## In the inst-sys

@wfeldt said if we have that dependency on our package it will automatically be honored for building the inst-sys.

### CODE REVIEW

File: package/libyui-qt-pkg.spec [hunk 1]
```diff
@@ -125,7 +128,7 @@ cmake .. \
         -DCMAKE_BUILD_TYPE=RELEASE
 %endif
 
-make %{?jobs:-j%jobs}
+make VERBOSE=1 %{?jobs:-j%jobs}
```
Comment (lslezak) [inline package/libyui-qt-pkg.spec]: Do we really need the verbose mode during the package build or was it just for debugging??

The point it is that the other libyui packages do not use it, it should be consistent...

Comment (shundhammer) [reply package/libyui-qt-pkg.spec]: Rudi complained badly when we tried to figure out together what's happening that we always force cmake/make to "silent" mode, so the exact command line that would have shown the linker command was not shown. So I added it upon his request.

Comment (shundhammer) [reply package/libyui-qt-pkg.spec]: Mid-term we should also add it to our other cmake-based packages IMHO.

Comment (lslezak) [reply package/libyui-qt-pkg.spec]: OK, thanks for the info!

**Comment by shundhammer** [comment]
It might be a result of this:

  https://stackoverflow.com/questions/28088100/gcc-build-links-but-shared-library-does-not-appear-with-ldd

    ld --as-needed

**Pull request merged**